### PR TITLE
Print state tool update version during an update to the latest version.

### PR DESCRIPTION
**Pull request by mitchell-as**
<table>
<td>
  <a href="https://activestatef.atlassian.net/browse/DX-1099" title="DX-1099" target="_blank">StoryDX-1099</a>  UPDATE: When run `state update` no version number provided like it done in install flow.
  </td></table>
  <br />

### CODE REVIEW

File: internal/runners/update/update.go [hunk 1]
```diff
@@ -64,6 +64,10 @@ func (u *Update) Run(params *Params) error {
 		return nil
 	}
 
+	if params.Version == "" {
+		u.out.Notice(locale.Tl("updating_version", "Updating State Tool to version {{.V0}}", up.Version))
+	}
```
Comment (MDrakos) [inline internal/runners/update/update.go:69]: I think we need to update the output on line 49 above rather than add another `Updating` message

**Review by MDrakos — CHANGES_REQUESTED** [review]

File: internal/runners/update/update.go (hunk 1 above)
Comment (mitchell-as) [reply internal/runners/update/update.go:69]: I thought about that, but if the update check takes some time, the user is left "hanging" and wondering what is going on until we get a version to print to the output. Or is this a non-issue?

Comment (mitchell-as) [reply internal/runners/update/update.go]: Never mind, you were on the right track. I've pushed an update to our messaging around updates.

**Review by MDrakos — APPROVED** [review]
Looks good!

**Pull request merged**